lib/jnpr/junos/utils/fs.py

```diff
@@ -268,30 +270,34 @@ def _decode(fs):
     # directory_usage - filesystem directory usage
     # -------------------------------------------------------------------------
 
-    def directory_usage(self, path="."):
+    def directory_usage(self, path=".", depth=0):
         """
         Returns the directory usage, similar to the unix "du" command.
 
-        :returns: approximate directory usage, including subdirectories, in bytes
+        :returns: dict of directory usage, including subdirectories if depth > 0
         """
-        rsp = self._dev.rpc.get_directory_usage_information(path=path, depth="0")
+        BLOCK_SIZE = 512
 
-        used_space = rsp.findtext("directory/used-space")
+        rsp = self._dev.rpc.get_directory_usage_information(path=path, depth=str(depth))
```

Comment:
@stacywsmith  using normalize=True was my initial intention, but then test test_directory_usage() fails with reason that is completely unclear to me:

```
Error
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/home/peter/PycharmProjects/py-junos-eznc/tests/unit/utils/test_fs.py", line 272, in test_directory_usage
    self.assertEqual(self.fs.directory_usage(path="/var/tmp"),
  File "/home/peter/PycharmProjects/py-junos-eznc/lib/jnpr/junos/utils/fs.py", line 281, in directory_usage
    rsp = self._dev.rpc.get_directory_usage_information(path=path, depth=str(depth), normalize=True)
  File "/home/peter/PycharmProjects/py-junos-eznc/lib/jnpr/junos/rpcmeta.py", line 156, in _exec_rpc
    return self._junos.execute(rpc, **dec_args)
  File "/usr/local/lib/python3.5/dist-packages/mock/mock.py", line 1062, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/mock/mock.py", line 1128, in _mock_call
    ret_val = effect(*args, **kwargs)
  File "/home/peter/PycharmProjects/py-junos-eznc/tests/unit/utils/test_fs.py", line 338, in _mock_manager
    device_params = kwargs['device_params']
KeyError: 'device_params'
```

I see other utility methods use strip() rather extensively - I guess this is probably due to the same reason.